src/main/resources/META-INF/rewrite/spring-boot-30.yml

```diff
@@ -200,3 +201,28 @@ recipeList:
       methodPattern: org.springdoc.core.models.GroupedOpenApi.Builder addOpenApiCustomiser(..)
       newMethodName: addOpenApiCustomizer
       matchOverrides: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.spring.boot3.MigrateDropWizardDependencies
+displayName: Migrate dropWizard dependencies to Spring Boot 3.x
+description: Migrate dropWizard dependencies to the new artifactId, since these are changed with Spring Boot 3.
+tags:
+  - spring
+  - boot
+  - dropwizard
+recipeList:
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: com.codahale.metrics.servlet
+      newPackageName: com.codahale.metrics.servlets
+  - org.openrewrite.java.ChangePackage:
+      oldPackageName: io.dropwizard.metrics.servlet
+      newPackageName: io.dropwizard.metrics.servlets
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: io.dropwizard.metrics
+      oldArtifactId: metrics-servlet
+      newArtifactId: metrics-servlets
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: io.dropwizard.metrics
+      oldArtifactId: metrics-jakarta-servlet
+      newArtifactId: metrics-jakarta-servlets
```

Comment:
The way I understand the issue these two recipes are mixed up. metrics-servlet is migrated to metrics-jakarta-servlet. Further, we might have to check on the dependency versions.

I suggest we add a test, so we can catch some of these errors.

Reply:
@knutwannheden @timtebeek once you confirm changes are ok, I will add a test.

Reply:
@knutwannheden Added Test as well, could you please review it.